docs/i18n.md

```diff
@@ -0,0 +1,143 @@
+# OSF Angular – Internationalization (i18n) Strategy
+
+## Index
+
+- [Overview](#overview)
+- [Integration: `@ngx-translate/core`](#integration-ngx-translatecore)
+- [Usage Guidelines](#usage-guidelines)
+- [Format: `en.json`](#format-enjson)
+- [Source of Truth: `en.json` Only](#source-of-truth-enjson-only)
+- [Language Branch Workflow](#language-branch-workflow)
+- [Summary](#summary)
+
+---
+
+## Overview
+
+The OSF Angular project uses [`@ngx-translate/core`](https://github.com/ngx-translate/core) to manage internationalization (i18n) across the entire application. This allows for consistent, dynamic translation of all user-visible text, making it easier to support multiple languages.
+
+All strings rendered to users—whether in HTML templates or dynamically via Angular components—must be sourced from the centralized i18n JSON files located in:
+
+**IMPORTANT**
+The OSF application must maintain 100% translation coverage, as it is a requirement of grant funding that supports a globally distributed user base.
+
+```
+src/app/i18n/
+```
+
+## Integration: `@ngx-translate/core`
+
+To support multilingual content, the following module is included globally:
+
+```ts
+import { TranslatePipe } from '@ngx-translate/core';
+```
+
+The translation service (`TranslateService`) is injected where necessary and used to load and access translations at runtime.
+
+## Usage Guidelines
+
+### 1. HTML Templates
+
+In templates, use the `translate` pipe:
+
+```html
+<h1>{{ 'home.title' | translate }}</h1>
+```
+
+### 2. Dynamic Component Strings
+
+For component logic, use the `TranslateService`:
+
+```ts
+private readonly translateService = inject(TranslateService);
+public title!: string;
+
+ngOnInit(): void {
+  this.title = this.translate.instant('home.title');
+}
+```
+
+Avoid hardcoded strings in both the template and logic. All user-facing text must be represented by a translation key from the JSON files in `src/app/i18n`.
+
+## Format: `en.json`
+
+The primary English translation file is located at:
+
+```
+src/app/i18n/en.json
+```
+
+The structure should be **namespaced by feature or component** for maintainability:
+
+```json
+{
+  "home": {
+    "title": "Welcome to OSF",
+    "subtTtle": "Your research. Your control."
+  },
+  "login": {
+    "email": "Email Address",
+    "password": "Password"
+  }
+}
+```
+
+Avoid deeply nested keys, and always use consistent camel-casing and key naming for reuse and clarity.
```

Comment:
May be worth mentioning that there is a [`common` section](https://github.com/CenterForOpenScience/angular-osf/blob/main/src/assets/i18n/en.json#L2) in the translation file that may be a good place to look for frequently used phrases to avoid making one-off duplicate translations